dwave/preprocessing/presolve/cypresolve.pyx

```diff
@@ -26,7 +28,7 @@ from dimod.cyutilities cimport ConstNumeric
 
 cdef class cyPresolver:
     def __init__(self, cyConstrainedQuadraticModel cqm, *, bint move = False):
-        self.variables = cqm.variables  # always a copy
```

Comment:
So this was a python reference before?

Reply:
Cython objects are treated as Python objects in terms of assignment operator.